custom_components/midea_ac_lan/config_flow.py

```diff
@@ -272,9 +266,10 @@ async def async_step_discovery(
                 if not self._already_configured(
                     str(device_id), device[CONF_IP_ADDRESS]
                 ):
-                    self.available_device[device_id] = (
-                        f"{device_id} ({self.supports.get(device.get(CONF_TYPE))})"
-                    )
+                    self.available_device[
+                        device_id
+                    ] = f"{device_id} ({self.supports.get(
```

Comment:
Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings

Reply:
I cannot wait to get CI test all the supported versions of python and HA... it's a mess otherwise...